share/smack/top.py

```diff
@@ -680,21 +678,21 @@ def smackd_output(corral_output):
 
 def main():
     try:
-        global ARGS
-        ARGS = arguments()
+        global smack_args
```

Comment:
Why would you replace more descriptive names (like `smack_args`) with less descriptive names (like just `ARGS`)?